core/include/cubos/core/geom/plane.hpp

```
@@ -5,11 +5,38 @@
 
 #include <glm/glm.hpp>
 
+#include <cubos/core/data/deserializer.hpp>
+#include <cubos/core/data/serializer.hpp>
+
 namespace cubos::core::geom
 {
     /// @brief Plane shape.
     struct Plane
     {
-        const glm::vec3 normal; ///< The normal of the plane.
+        glm::vec3 normal{0.0f}; ///< The normal of the plane.
```

Should not default to an invalid value - set it to face Y+ by default.